### PR TITLE
MVC-Rails

### PR DESCRIPTION
**Pull request by Addisu87**
# I have created a Ruby on Rails application that allows the user to:
- register and log in, so that the data is private to them.
- introduce new transactions associated with a category.
- see the money spent on each category.
- Loom link: https://www.loom.com/share/eb1e2d8e334749b48f58e8634dd3e60f

### CODE REVIEW

File: app/controllers/purchases_controller.rb [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @purchase.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /purchases/1 or /purchases/1.json
+  def destroy
+    @category = Category.find(params[:category_id])
```
Comment (rustam-tolipov) [inline app/controllers/purchases_controller.rb:54]: - [ ] You did good job. However, I cannot make it work your `delete` button. Kindly make it functional.


https://user-images.githubusercontent.com/69011963/195844472-6c9fc299-f0bb-4de0-9ddb-15c0929b4e7c.mov

File: app/assets/stylesheets/application.tailwind.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
+/* stylelint-disable */
+
```
Comment (rustam-tolipov) [inline app/assets/stylesheets/application.tailwind.css:2]: You did a good job. But, somehow, your design doesn't meet the original design. So, I would suggest you maybe give some attention to styling.

File: README.md [hunk 3]
```diff
@@ -64,12 +86,11 @@ Give a ⭐️ if you like this project!
...
+- Original design idea by Gregoire Vella on Behance
+- My supportive family 😊
+- Microverse
 - Hat tip to anyone whose code was used
-- Inspiration
-- etc
 
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (rustam-tolipov) [inline README.md:96]: [OPTIONAL] Kindly add MIT license to your project.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @Addisu87, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Great Design_


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Addisu87** [comment]
@Rustamxon7, I will try to fix the delete buttons, but I think they are not required to implement in this project. I have added that as an additional. Maybe I will remove it.

**Review by dasileker — APPROVED** [review]
### **Hi @Addisu87,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**